### DISCUSSION TITLE
[BUG]Pending Wallet Transaction from 1 week!

### DISCUSSION THREAD

**Discussion by NavaChia** [Chia-Network/chia-blockchain#5337] [Support]
**Describe the bug**
Hi!, I touched the 2 coins and I tried to send it to my Okex Wallet, and it has been in pending status for 1 week, in the rate indicate "0" due to ignorance of how the system worked, is there anything that can be done? in the end it comes even late? Or is it possible that I lost my 2 coins when sending to my own Wallet? Thank you very much, please hope someone can help me.

**To Reproduce**

Send my 2 coins to my okex wallet putting rate "0".
Pending Wallet Transaction from 1 week!.

**Expected behavior**
That it had been correctly sent to my okex wallet

**Screenshots**
![image](https://user-images.githubusercontent.com/84290093/118496693-a6cb9e80-b724-11eb-8493-e3d516a2273e.png)
![image](https://user-images.githubusercontent.com/84290093/118496714-ab905280-b724-11eb-82c4-13a5f00cd224.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Chia BlockChain 1.1.5

**Additional context**

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
